### ISSUE TITLE
Theme inheritance

### ISSUE THREAD

**Issue by arnaudvergnet** [coppolaemilio/dialogic#232]
**Is your feature request related to a problem? Please describe.**
Sometimes we just want to make a small variation of our base theme (eg: change only the font when different characters talk). Having a base theme we can inherit from would greatly simplify this process.

**Describe the solution you'd like**
For each theme, select the parent theme to use. Any property changed in the child theme will override the parent. We could then have a parent-child chain, which could become hard to parse on run.

**Describe alternatives you've considered**
Right now we can simulate this by duplicating the base theme. But this can become really hard to maintain if we want to edit the base theme: we then need to recreate every theme derived from it.

**Comment by coppolaemilio** [comment]
I think we can close this one since themes are already Godot themes and you can create custom scenes and inherit whatever resource/scene you want.
